src/main/java/meditracker/storage/FileReaderWriter.java

```diff
@@ -119,4 +130,58 @@ public static void saveMediTrackerData(MedicationManager medManager) {
             System.out.println(e.getMessage());
         }
     }
+
+    /**
+     * Loads the daily medication information from a fixed file data/dailymed/today.txt.
+     * The loading functionality will be improved in v2.0.
+
+     * @param dailyMedData A list of type String for the daily medication data.
+     * @throws FileReadWriteException if there is an issue creating the file.
+     */
+    public static void saveDailyMedicationData(List<String> dailyMedData) throws FileReadWriteException {
```

Comment:
is the JavaDoc (line 135) and saveDailyMedicationData() name swapped

Reply:
Ah yes, thanks for the catch. Must have swapped the javadoc for the 2 functions around during committing. Fixed in cbca7dfa7748336954d21dfcbd4d7911f48f34c7